docs/src/man/getting_started.md

```diff
@@ -45,7 +45,7 @@ julia> df = DataFrame(A = 1:4, B = ["M", "F", "F", "M"])
 
 ```
 
-Columns can be directly (i.e. without copying) accessed via `df.col`, `df."col"`, `df[!, :col]` or `df[!, "col"]`. The two latter syntaxes are more flexible as they allow passing a variable holding the name of the column, and not only a literal name. Note that column names can be either symbols (written as `:col`, `:var"col"` or `Symbol("col")`) or strings (written as `"col"`).
+Columns can be directly (i.e. without copying) accessed via `df.col`, `df."col"`, `df[!, :col]` or `df[!, "col"]`. The two latter syntaxes are more flexible as they allow passing a variable holding the name of the column, and not only a literal name. Note that column names can be either symbols (written as `:col`, `:var"col"` or `Symbol("col")`) or strings (written as `"col"`; in particular variable interpolation using `$` is not allowed).
```

Comment:
This isn't correct for `df[!, col]`.

TBH I'm not sure this really needs to be mentioned here, this is the "getting started" page.

Reply:
right - `$` is disallowed only in `getproperty`/`setproperty!`. @oxinabox - are you OK to skip commenting on this? (in general the behaviour is not DataFrames.jl related but just how Julia Base works)

Reply:
I think it is important.
this is going to to be the first time most people encounter a string in this position.
And I would naturally assume that interpolation would work in this string unless told otherwise.

Reply:
Well Julia will tell them otherwise quite soon if they think it works... :-)

Reply:
I think we can keep it. I have also added a comment that interpolation does not work for `:var"a$i"`.